tt.go

```diff
@@ -0,0 +1,282 @@
+package tupleconv
+
+import (
+	"fmt"
+)
+
+// TypeName is the data type for type names.
+type TypeName string
+
+// Types are supported tarantool types.
+const (
+	TypeBoolean   TypeName = "boolean"
+	TypeString    TypeName = "string"
+	TypeInteger   TypeName = "integer"
+	TypeUnsigned  TypeName = "unsigned"
+	TypeDouble    TypeName = "double"
+	TypeNumber    TypeName = "number"
+	TypeDecimal   TypeName = "decimal"
+	TypeDatetime  TypeName = "datetime"
+	TypeUUID      TypeName = "uuid"
+	TypeArray     TypeName = "array"
+	TypeMap       TypeName = "map"
+	TypeVarbinary TypeName = "varbinary"
+	TypeScalar    TypeName = "scalar"
+	TypeAny       TypeName = "any"
+	TypeInterval  TypeName = "interval"
+)
+
+const (
+	defaultThousandSeparators = ""
+	defaultDecimalSeparators  = "."
+	defaultNullValue          = ""
+)
+
+// TypeToTTConvFactory is a factory capable of creating converters from Type
+// to tarantool types.
+type TypeToTTConvFactory[Type any] interface {
+	// MakeTypeToBooleanConverter creates a converter from Type to boolean.
+	MakeTypeToBooleanConverter() Converter[Type, any]
+
+	// MakeTypeToStringConverter creates a converter from Type to string.
+	MakeTypeToStringConverter() Converter[Type, any]
+
+	// MakeTypeToUnsignedConverter creates a converter from Type to unsigned.
+	MakeTypeToUnsignedConverter() Converter[Type, any]
```

Comment:
At least it's shorter, but it's even clearer.
```suggestion
	// BooleanConverter creates a converter from Type to boolean.
	(Get?)BooleanConverter() Converter[Type, any]

	// StringConverter creates a converter from Type to string.
	StringConverter() Converter[Type, any]

	// UnsignedConverter creates a converter from Type to unsigned.
	UnsignedConverter() Converter[Type, any]
	// etc
```

Reply:
Changed.